docs/documentation/release_notes/topics/21_1_0.adoc

```diff
@@ -9,3 +22,25 @@ The Account Console version 3 is now available as an experimental feature in Key
 ```
 bin/kc.sh start-dev --features=account3
 ```
+= Changes to Keycloak Authorization Services support in Keycloak Java-based Adapters
+
+As part of the removal of the https://www.keycloak.org/2023/03/adapter-deprecation-update[deprecated] adapters, the Keycloak Policy Enforcer was extracted from the adapters code base
+into a separate dependency:
+
+<dependency>
+    <groupId>org.keycloak</groupId>
+    <artifactId>keycloak-policy-enforcer</artifactId>
+    <version>${version.keycloak}</version>
+</dependency>
```

Comment:
I believe we can get some [syntax highlighting](https://docs.asciidoctor.org/asciidoctor/latest/syntax-highlighting/) with this as well:
```suggestion
[source,xml]
----
<dependency>
    <groupId>org.keycloak</groupId>
    <artifactId>keycloak-policy-enforcer</artifactId>
    <version>${version.keycloak}</version>
</dependency>
----
```